src/components/select-field/messages.js

```diff
@@ -6,6 +6,11 @@ const messages = defineMessages({
         description: 'text shown on the Clear All option in the options list',
         id: 'boxui.selectField.clearAll',
     },
+    searchPlaceholder: {
+        defaultMessage: 'Search',
```

Comment:
I feel like we must have this as a common string somewhere already...

Reply:
export const VIEW_SEARCH: 'search' = 'search';

We do have this but it doesn't look like it's the right search constant that we should be using.

Reply:
No I mean something like `boxui.contentExplorer.searchPlaceholder`. Unfortunately its not in a common file so it can't be used as-is.

Reply:
Ah got it.